### PR TITLE
Add is_superuser property to analytics events

### LINKED ISSUE #5211: Add superuser property to analytics events
Backend analytics user properties should include the information if the respective (multi)user is superuser

### PR DESCRIPTION
**Pull request by fm3**
Superusers (i.e. employees) are noise for analytics, so it may be desirable to filter out their events.

### Steps to test:
- in application.conf set `backendAnalytics.verboseLoggingEnabled = true`, open a dataset
- in analytics logging, the is_superuser property should show (and be true iff the requesting user is a superuser)

### Issues:
- fixes #5211

------
- [x] Ready for review

### CODE REVIEW

File: app/models/analytics/AnalyticsService.scala [hunk 1]
```diff
@@ -43,10 +43,19 @@ class AnalyticsService @Inject()(rpc: RPC, wkConf: WkConf, analyticsLookUpServic
...
 }
 
-class AnalyticsLookUpService @Inject()(userDAO: UserDAO, wkConf: WkConf) extends LazyLogging {
+class AnalyticsLookUpService @Inject()(userDAO: UserDAO, multiUserDAO: MultiUserDAO, wkConf: WkConf)
+    extends LazyLogging {
+  implicit val ctx: DBAccessContext = GlobalAccessContext
+
+  def isSuperUser(userId: ObjectId): Fox[Boolean] =
+    for {
+      user <- userDAO.findOne(userId)
```
Comment (youri-k) [inline app/models/analytics/AnalyticsService.scala:52]: Do we really need to take the detour with the `userId`?
I think we can directly pass the `multiUserId` and save one db call.

**Review by youri-k — CHANGES_REQUESTED** [review]
See my comment about having to look up the user again. I feel there is optimization potential.
Otherwise LGTM :slightly_smiling_face:

File: app/models/analytics/AnalyticsService.scala (hunk 1 above)
Comment (fm3) [reply app/models/analytics/AnalyticsService.scala]: Absolutely right! Good catch :) Could you have a look again?

**Review by youri-k — APPROVED** [review]
LGTM :shipit:

**Pull request merged**